app/cdap/components/CreateRecipe/index.tsx

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
CreateRecipe component also can be moved to RecipeManagement folder

Reply:
Moved CreateRecipe component to RecipeManagement folder